src/style.css

```diff
@@ -1,3 +1,74 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  --hilight-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: flex;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.nav-link {
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  font-family: Roboto, sans-serif;
```

Comment:
From the checklist: 'Remember to use fallback fonts - alternative font-family in case the main one doesn't work'. For example, you can use a more generic font like Arial as a fallback.